training/src/test/java/wolox/training/controllers/BookControllerTest.java

```diff
@@ -28,11 +33,14 @@
 import static wolox.training.utils.Utils.BODY_WITH_BOOK_UPDATED;
 
 @RunWith(SpringRunner.class)
-@WebMvcTest(BookController.class)
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
 public class BookControllerTest {
```

Comment:
Add tests in order to check that a "403 Forbidden" is thrown when a non-authenticated user is trying to get to a closed endpoint.